code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -397,4 +397,43 @@ emp_act
 	if(check_head && head && (head.flags_cover & HEADCOVERSEYES))
 		return TRUE
 	if(check_mask && wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH))
-		return TRUE
\ No newline at end of file
+		return TRUE
+
+/mob/living/carbon/human/attackby(obj/item/I, mob/user, params)

```

Comment:
Don't make this a `human/attackby` 

make this an `organ/external/attack`

Reply:
🤔

Reply:
@tigercat2000 I feel this is actually less snowflake because having huge `if` statements in human attacky  gets messy, and is more prone to interference on intended behavior, especially since @uraniummeltdown's attacking refactor.

Reply:
I think that could make sense, too. One thing in favor of tiger's view on this is that the organ's methods/properties are only 4 times in this method, and all the rest of the accesses are to stuff on human. Also this is the only thing in human/attackby right now.

Reply:
@Fox-McCloud I think it's better off on humans because it's not applicable to other carbons and there's a ton of human var accesses in the block - more than organ accesses. It's better off for the mob that contains the species to determine the behavior of things involved with having that species than have snowflake checks elsewhere, like organ code, to do stuff based on the species of the mob.

Reply:
Coin flip for a tie breaker?

Reply:
Esh. I can see why you would want to put it in organ/external/attack, but honestly... I can't decide. So.. I'll leave that option to you all and keep my opinions over here :P (though I can see +'s to both :/)

Reply:
@Crazylemon64 thoughts?

Reply:
Coin flip, gentlemen?

Reply:
@Crazylemon64 this PR was already approved in #8783, but please let me know if your dislike from a design POV will change this, so I know whether I should implement your requested fixes.

Reply:
Did you guys ever figure out where you wanted this code? Seems absurd to me that this has been open for 5 months without an actual decision (and subsequent merge/close.)